wrappers/services_test.go

```diff
@@ -115,7 +115,7 @@ func (s *servicesTestSuite) addSnapServices(snapInfo *snap.Info, preseeding bool
 
 func (s *servicesTestSuite) TestAddSnapServicesAndRemove(c *C) {
 	info := snaptest.MockSnap(c, packageHello, &snap.SideInfo{Revision: snap.R(12)})
-	svcFile := filepath.Join(s.tempdir, "/etc/systemd/system/snap.hello-snap.svc1.service")
```

Comment:
Thank you for the review! When I first did the change to append the root directory to the `snapHomeDirs` values, I noticed that one unit test was failing (I don't remember exactly which one). The cause was that the value of `s.tempdir` and `dirs.GlobalRootDir` were different and caused `dirs.SetSnapHomeDirs()` to not detect the root directory since the path passed to `dirs.SetSnapHomeDirs()` was of the form `<s.tempdir>/rest/of/path`. 

After seeing this and fixing the issue, I decided to replace the instances of `s.tempdir` in unit tests with `dirs.GlobalRootDir` for consistency. Using `dirs.GlobalRootDir` should also make it easier to catch bugs in unit tests where `s.tempdir` could be modified without calling `dirs.SetRootdir()` on the new value.

Reply:
Thanks, this makes sense.